LogicalTypes.md

```diff
@@ -684,44 +702,68 @@ optional group my_list (LIST) {
 }
 ```
 
-Some existing data does not include the inner element layer. For
-backward-compatibility, the type of elements in `LIST`-annotated structures
-should always be determined by the following rules:
+##### 2-level structure
+
+Some existing data does not include the inner element layer, resulting in a
+`LIST` that annotates a 2-level structure. Unlike the 3-level structure, the
+repetition of a 2-level structure can be `optional`, `required`, or `repeated`.
+When it is `repeated`, the `LIST`-annotated 2-level structure can only serve as
+an element within another `LIST`-annotated 2-level structure.
+
+```
+<list-repetition> group <name> (LIST) {
+  repeated <element-type> <element-name>;
+}
+```
+
+For backward-compatibility, the type of elements in `LIST`-annotated structures
```

Comment:
Explicitly add `if they cannot be determined as 3-level structures` to partially address the comment on how to resolve a list-annotated group in practice.

Reply:
So here following the rule:
1. Trying to parse as 3-level structure
2. Trying to parse 2-level legacy
3. Trying to parse 1-level legacy

?

Reply:
I agree with @etseidl. This section only discusses LIST-annotated structures.